### PR TITLE
[B-7] Currency convertion

### PR DESCRIPTION
**Pull request by DanwerOK**

### CODE REVIEW

File: src/main/java/com/ponodan/movieland/dao/JdbcConfig.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    @Value(value = "${jdbc.url}")
+    private String dbUrl;
+
+    @Value(value = "${jdbc.username}")
+    private String dbUser;
+
+    @Value(value = "${jdbc.password}")
+    private String dbPassword;
+
+    @Value(value = "${jdbc.connectionPoolSize}")
```
Comment (Rooman) [inline src/main/java/com/ponodan/movieland/dao/JdbcConfig.java]: @Value("${jdbc.connectionPoolSize}")

File: src/main/java/com/ponodan/movieland/service/impl/CurrencyServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        this.currencyDao = currencyDao;
+    }
+
+    @Override
+    public void changeMoviePrice(Movie movie, Enum currency){
+        Map<Enum, Double> currencyMap = currencyDao.getAll();
+
+        Double rate = currencyMap.get(currency);
+        Double currentPrice = movie.getPrice();
+        Double priceInCurrency = currentPrice / rate;
```
Comment (Jokeroff) [inline src/main/java/com/ponodan/movieland/service/impl/CurrencyServiceImpl.java]: round

File: src/main/java/com/ponodan/movieland/web/controller/ParamsValidator.java [hunk 3]
```diff
@@ -27,11 +28,11 @@
         Map<Enum, Enum> validationResult = new LinkedHashMap<>();
 
         for (Map.Entry<String, String> param: params.entrySet()) {
-            Enum paramName = SortingColumn.getByName(param.getKey());
+            Enum paramName = UrlParam.getByName(param.getKey());
             Enum paramValue = SortingOrder.getByName(param.getValue());
 
-            if (((paramName == SortingColumn.PRICE) && SortingOrder.contains(paramValue))
-                    || ((paramName == SortingColumn.RATING) && (paramValue == SortingOrder.DESC))
+            if (((paramName == UrlParam.PRICE) && SortingOrder.contains(paramValue))
```
Comment (Jokeroff) [inline src/main/java/com/ponodan/movieland/web/controller/ParamsValidator.java]: "&& SortingOrder.contains(paramValue)" possibly redundant

File: src/main/java/com/ponodan/movieland/web/controller/ParamsValidator.java [hunk 4]
```diff
@@ -45,4 +46,35 @@
 
         return validationResult;
     }
+
+    static Enum validateMoviePriceCurrencyParams(Map<String, String> params){
+        if (params.size() > 1){
```
Comment (Jokeroff) [inline src/main/java/com/ponodan/movieland/web/controller/ParamsValidator.java]: Map<String, String> can be logged directly

**Pull request merged**